transform/emr/emr_launch.py

```
@@ -0,0 +1,75 @@
+    Steps=[
+        {
+            "Name": "Run Spark Job",
+            "ActionOnFailure": "CONTINUE",
+            "HadoopJarStep": {
+                "Jar": "command-runner.jar",
+                "Args": [
+                    "spark-submit",
+                    "--deploy-mode", "cluster",
+                    "--conf", f"spark.executorEnv.OPENAI_API_KEY={os.getenv('OPENAI_API_KEY')}",
+                    "--conf", f"spark.driver.extraJavaOptions=-DOPENAI_API_KEY={os.getenv('OPENAI_API_KEY')}",
+                    s3_script_path,
```

Ruff (0.8.2)

Undefined name `s3_script_path`

(F821)